apis/beacon/pool/bls_to_execution_changes.yaml

```diff
@@ -23,26 +23,54 @@ get:
 post:
   operationId: submitPoolBLSToExecutionChange
   summary: Submit SignedBLSToExecutionChange object to node's pool
-  description: Submits SignedBLSToExecutionChange object to node's pool and if passes validation node MUST broadcast it to network.
+  description: Submits a list of SignedBLSToExecutionChange objects to node's pool. Any that pass validation MUST be broadcast to the network.
   tags:
     - Beacon
   requestBody:
     required: true
     content:
       application/json:
         schema:
-          $ref: '../../../beacon-node-oapi.yaml#/components/schemas/SignedBLSToExecutionChange'
+          type: array
+          items:
+            $ref: '../../../beacon-node-oapi.yaml#/components/schemas/SignedBLSToExecutionChange'
   responses:
     "200":
-      description: BLS to execution change is stored in node and broadcasted to network
+      description: "An array of results, presented in the same order as the submission"
+      content:
+        application/json:
+          schema:
+            title: submitPoolBLSToExecutionChangeResponse
+            type: object
+            required: [data]
+            properties:
+              data:
+                type: array
+                description: The result of each blsToExecutionChange submitted, in order of submission
+                items:
+                  type: object
+                  required: [status]
+                  properties:
+                    status:
+                      type: string
+                      description: |
+                        - accepted: The bls_to_execution_change was accepted, placed in the local pool, and broadcast to the network
+                        - error: The bls_to_execution_change was not accepted. More context may be supplied in the associated message.
+                      enum:
+                        - accepted
+                        - error
+                      example: accepted
+                    message:
+                      type: string
+                      description: error message if status == error
```

Comment:
Maybe we should model this after the other batched pool endpoint rather than the key-manager API?

Some advantages of consistency with the `/pool/attestations` API:

- User gets an error status code if any of the messages fail. I think this is slightly clearer/more obvious when submitting manually, and could work well with `curl --fail-with-body` in simple scripts, etc. The failed messages are still in the error response in case of error.
- Consistency makes the API (slightly) easier to use and code against.

Reply:
Fair point. keymanager is what I had working yaml for so it was easier, but I can have a look at the attestation pool and refactor closer to that...